src/nunit.analyzers/Extensions/ArgumentListSyntaxExtensions.cs

```diff
@@ -36,5 +51,37 @@ internal static class ArgumentListSyntaxExtensions
 
             return @this.WithArguments(newSeparatedList);
         }
+
+        private static bool TryGetFirstEndOfLineTrivia(SyntaxToken openParenToken, SyntaxToken[] separators, out SyntaxTrivia trailingTrivia)
+        {
+            // If there's only one argument, there are no separators.
+            // Therefore, use the trailing trivia of the opening parenthesis into account to make our best guess.
+            if (separators.Length == 0)
+            {
+                foreach (var trivia in openParenToken.TrailingTrivia)
+                {
+                    if (trivia.IsKind(SyntaxKind.EndOfLineTrivia))
+                    {
+                        trailingTrivia = trivia;
+                        return true;
+                    }
+                }
+            }
+
+            foreach (var separator in separators)
```

Comment:
Could add an `else`  here to prevent creating an enumerator for an array with no elements.